Dockerfile.distroless

```
@@ -0,0 +1,49 @@
+ARG ARCH="amd64"
+ARG OS="linux"
+
+# Stage: get CA certs and tzdata from a small Debian image
+FROM debian:trixie-slim AS certs
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates tzdata \
+    && rm -rf /var/lib/apt/lists/*
+
+# Final minimal image based on distroless static
+FROM gcr.io/distroless/static:nonroot
+
```

The parameters need to be added to this stage
```suggestion
ARG ARCH
ARG OS
```